roadmap/implementers-guide/src/runtime/shared.md

```diff
@@ -20,8 +20,14 @@ pub(crate) const SESSION_DELAY: SessionIndex = 2;
 ## Storage
 
 ```rust
-// The current session index within the Parachains Runtime system.
+/// The current session index within the Parachains Runtime system.
 CurrentSessionIndex: SessionIndex;
+/// All the validators actively participating in parachain consensus.
+/// Indices are into the broader validator set.
+ActiveValidatorIndices: Vec<ValidatorIndex>,
+/// The parachain attestation keys of the validators actively participating in parachain consensus.
+/// This should be the same length as `ActiveValidatorIndices`.
```

Comment:
we could store it as a `Vec` of pairs to enforce this, also it's a bit unexpected that tests set the validator keys, but not indices, which may result in some unexpected errors when running tests

Reply:
A `Vec` of pairs is just more unwieldy to use and may require additional allocation within the block execution